scripts/nbd-firewall-config.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems. All rights reserved.
+
+set -e
+
+##############################################################
+# Use this script to open/close the specified interfaces
+# (IP addresses) for incoming TCP connections to the NBD port.
+#
+# Usage:
+#   ./nbd-firewall-config set [address...]
```

Comment:
How often is this script called ?

Reply:
Rarely.
Only when a pool administrator changes the configuration, specifying a different set of networks that should be used for NBD. Then on each host, xapi will work call this script specifying the addresses of all its interfaces that are on those networks.